### PR TITLE
Designer tables and created Admin, Session, Newsfeed tables

### PR DESCRIPTION
**Pull request by ehsan2793**
Updated Db. Designer tables and created Admin, Session, Newsfeed tables 
the old Db. The designer was missing some of the tables; I used the tables created in code to create this new table design.

new address => [Db.Designer](https://app.dbdesigner.net/designer/schema/480102)

The `newsfeed` table is not connected to any other table, and the last group did not use it. we can ask the app owner about its use so we can have a better understanding of the app

loom video: https://www.loom.com/share/095137aec1cb4728bc7aa05d8dd43b6c

### CODE REVIEW

**Comment by thi-hong-van-nguyen** [comment]
Great work!!
For some reason, when I click on the link to dbdesigner, it redirects me to https://app.dbdesigner.net/designer. >.<

**Comment by ehsan2793** [comment]
Thanks, @vannguyen141290. I will look into it see why that's happening

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <img src="https://user-images.githubusercontent.com/65091914/142253036-074f454e-62ab-445d-a30c-26e371e76e0a.png">
-Source: https://app.dbdesigner.net/designer/schema/0-codeheroes-99ad6258-244f-484c-a12f-151cb4aec1b2
+Source: https://app.dbdesigner.net/designer/schema/0-codeheroes-99ad6258-244f-484c-a12f-151cb4aec1b2  ===> `old table`
+
+Updated table Source: https://app.dbdesigner.net/designer/schema/480102
```
Comment (AshleyCoder3) [inline README.md:4]: Link re-directs me also to just https://app.dbdesigner.net/designer

**Review by AshleyCoder3 — CHANGES_REQUESTED** [review]
You may have to change permissions on your DB

File: README.md (hunk 1 above)
Comment (ehsan2793) [reply README.md:4]: that's so weird. Can you try this address [https://dbdesigner.page.link/WTZRbVeTR7EzLvs86](https://dbdesigner.page.link/WTZRbVeTR7EzLvs86)
and please let me know if it works

Comment (AshleyCoder3) [reply README.md:4]: that one works now thanks so much

**Pull request closed without merging**